contracts/external/cw-payroll/src/msg.rs

```diff
@@ -1,41 +1,32 @@
-use crate::state::GenericBalance;
 use cosmwasm_schema::{cw_serde, QueryResponses};
-use cosmwasm_std::Uint128;
-use cw20::Cw20ReceiveMsg;
+use cw20::{Cw20ReceiveMsg};
+
+use crate::balance::WrappedBalance;
 
 #[cw_serde]
 pub struct InstantiateMsg {
     pub admin: Option<String>,
 }
+pub type StreamId = u64;
+
 
 #[cw_serde]
 pub enum ExecuteMsg {
     Receive(Cw20ReceiveMsg),
     Distribute {
-        id: u64, // Stream id
+        id: StreamId, // Stream id
     },
-    // TODO: Add this! :D
-    // TODO: Only called by stream admin
-    // NOTE: Pauses a stream (paused: true), 
-    //       vesting is stopped
-    //       distribute of unlocked funds should be allowed (based on paused_time)
     PauseStream {
-        id: u64, // Stream id
+        id: StreamId, // Stream id
     },
-    // TODO: Add this! :D
-    // TODO: Only called by stream admin
-    // NOTE: Removes any pause from a stream, 
-    //       optionally can shift the start/end time if needed for vesting flexibility
     ResumeStream {
-        id: u64, // Stream id
+        id: StreamId, // Stream id
         start_time: Option<u64>,
```

Comment:
Let's remove `start_time` & `end_time` here. I can think of ways that the admin could be mean for shifting agreed upon params. Better to just have the stream get removed & recreated than cover edge cases on this.